tests/storage/fsp/test_fsp_sr.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import os
+import logging
+from lib.common import wait_for, vm_image
+from lib.vdi import VDI
+
+DIRECTORIES_PATH = 'directories'
```

Comment:
```suggestion
import pytest
import os
import logging

from lib.common import wait_for, vm_image
from lib.vdi import VDI

from .conftest import DIRECTORIES_PATH
```

For this to work properly you will also need to create an empty `__init__.py` file in `tests/storage/fsp/`.